lib/ClangImporter/DWARFImporter.cpp

```diff
@@ -15,6 +15,11 @@
 
 using namespace swift;
 
+// vtable anchor.
+void DWARFImporterDelegate::lookupValue(
+    StringRef name, llvm::Optional<ClangTypeKind> kind,
+    SmallVectorImpl<clang::Decl *> &results) {}
```

Comment:
Typically we do this with a dummy function, so that clients can still get the benefit of inlining super calls (or seeing that they don't need to). Example:

https://github.com/apple/swift/blob/bf2a4712ecafbcaa364579dd762e3f7398331bf9/include/swift/AST/TypeWalker.h#L41